components/resources/wallet_strings.grdp

```diff
@@ -618,6 +618,23 @@
   <message name="IDS_BRAVE_WALLET_SELL_WITH_PROVIDER" desc="Sell with Provider option label">Sell with <ph name="OFFRAMP_PROVIDER">$1<ex>Ramp</ex></ph></message>
   <message name="IDS_BRAVE_WALLET_BUY_WITH_SARDINE" desc="Buy with Sardine option label">Buy with Sardine</message>
   <message name="IDS_BRAVE_WALLET_BUY_DISCLAIMER" desc="Brave disclaimer text in buy screen">Financial and transaction data is processed by our onramp partners. Brave does not collect or have access to such data.</message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTIONS_PARTNER" desc="Transactions partner modal title">Transactions partner</message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_PARTNER_CONSENT" desc="Meld consent description">Buying or selling crypto in Brave Wallet uses Meld.io — an on-ramp/off-ramp aggregator that provides a smooth experience and the best available pricing. Your information will be shared with Meld.io to complete the transaction.</message>
```

Comment:
Should we be making the claim about the best available pricing? These types of things usually come with a disclaimer.

Reply:
cc: @jamesmudgett @kleantzogu

Reply:
@mkarolin we have updated this copy here https://github.com/brave/brave-core/compare/3f29b5d103ed154938a55caad7d0dd7726744356..def15c35ca71d06f687e85880eff6d9a11f7153b